conanfile.py

```diff
@@ -67,9 +67,8 @@ def requirements(self):
         if self.options.with_benchmark:
             self.requires("benchmark/1.8.3")
 
-    # Let people provide their own CMake for now
-    # def build_requirements(self):
-    #     self.tool_requires("cmake/3.29.0")
+    def build_requirements(self):
+        self.tool_requires("cmake/3.27.9")
```

Comment:
That's not the right move IMHO.

You should configure your conan global.conf./Profile options to pass CMAKE_MINIMUM_REQUIRED_VERSION is that's needed on mac intel, assuming that's what you're trying to fix (bzip2).

Or just do what I'm doing here:

export CMAKE_POLICY_VERSION_MINIMUM=3.5

https://github.com/jmarrec/conan-recipes/pull/12#pullrequestreview-3528899247